css/style.css

```diff
@@ -67,11 +66,155 @@ body {
 }
 
 header {
+   background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+   height:500px;
+   padding: 25px 90px 0px 90px;
+}
+
+header h1 {
+  font-weight: 800;
+  font-size: 30px;
+  color:white;
+}
+
+header h2 {
+  font-size: 30px;
+  font-weight:700;
+  margin-top:65px;
+  color:white;
+  text-align: center;
+}
+
+.getbutton {
+  border-radius: 5px;
+  font-weight: 700;
+  background-color: yellow;
+  width: 220px;
+  height:50px;
+  font-size: 20px;
+  margin-top:40px;
+  margin-left:310px;
+  padding: 10px 40px 14px 40px;
+  text-align: center;
+}
+
+.benefits {
+  padding: 50px 80px 50px 100px;
+  background-color: #f0efef;
+  clear:both;
+}
+
+.benefits h1 {
+  font-size:32px;
+  margin-bottom: 25px;
+  font-weight: 700;
+  color:#033048;
+}
+
+.benefits p {
+  font-size: 22px;
+  margin-bottom: 25px;
+  width:500px;
+  font-weight: 400;
+  color: #606161;
+}
+
+ul {
+  color:#606161;
+  font-size: 18px;
+  font-weight: 350;
+  margin-left: 85px;
+  width:380px;
+  list-style: inherit;
+}
+
+.test {
+  clear:both;
+  height:350px;
+  width:100%;
+  background: url(../images/testimonial_bg.jpg) no-repeat center top;
+}
+
+.quote {
+  font-weight: 100;
+  font-style: italic;
+  padding: 60px 300px 0px 300px;
+  color: white;
+  font-size: 20px;
+  text-align: center;
+  width: 400px;
+
+
+}
+
+.name{
+  font-size: 20px;
+  text-align: center;
+  font-weight: 700;
+  color:white;
+  padding: 10px 300px 0px 300px;
+  font-style: italic;
+}
+
+.yellowsec {
+  clear:both;
+  height:55px;
+  background-color: #f9e42e;
+  padding:33px;
+  margin-top:50px;
+  padding-top: 12px;
+}
 
+.signup {
+  height: 40px;
+  width: 160px;
+ padding-top: 5px;
+ margin-left: 400px;
```

Comment:
@cliffjadoo you need to add  `display: block` because a `<button>` is an inline element. Then change margin-left to  `margin-left: auto` to center the button.